solenspipe/solenspipe.py

```diff
@@ -343,15 +418,65 @@ def get_kmap(self,channel,seed,lmin,lmax,filtered=True):
         return self.cache[seed][:3] if filtered else self.cache[seed][3:]
 
     def get_mv_kappa(self,polcomb,talm,ealm,balm):
+        
         # Wrapper for qfunc
         return self.qfunc(polcomb,[talm,ealm,balm],[talm,ealm,balm])
 
     def qfunc(self,alpha,X,Y):
         # Wrapper for the core falafel full-sky lensing reconstruction function
         polcomb = alpha
-        return qe.qe_all(self.px,lambda x,y: self.theory.lCl(x,y),
+        return qe.qe_all(self.px,lambda x,y: self.theory.lCl(x,y),lambda x,y:self.theory_cross.lCl(x,y),
+                         self.mlmax,Y[0],Y[1],Y[2],estimators=[polcomb],
+                         xfTalm=X[0],xfEalm=X[1],xfBalm=X[2])[polcomb][0]
+
+    def qfunc_bh(self,alpha,X,Y,ils,blens,bhps,Alpp,A_ps):
+        polcomb=alpha
+        Tcmb = 2.726e6
```

Comment:
should this be hardcoded? @qujia7 @msyriac

Reply:
Some of the Bias hardening code requires using functions from cmblensplus from Toshiya. The Tcmb factor here is to address the fact in Toshiya's code the temperature is dimensionless whereas in solenspipe it is in microKelvins. So one don't expect this number to change.